src/selectors/breakpointAtLocation.js

```diff
@@ -0,0 +1,60 @@
+import { getSelectedSource } from "../reducers/sources";
+import { getBreakpoints } from "../reducers/breakpoints";
+import { isGeneratedId } from "devtools-source-map";
+
+function isGenerated(selectedSource) {
+  const sourceId = selectedSource.get("id");
+  return isGeneratedId(sourceId);
+}
+
+function getColumn(column, selectedSource) {
```

Comment:
I like this pattern better than keeping the selectors in the reducer. maybe we could have a folder structure here:
```
selectors/
|- breakpoints/
  |- index.js
  |- atLocation.js
```